libmachine/host/host.go

```diff
@@ -134,11 +135,13 @@ func (h *Host) Upgrade() error {
 
 	provisioner, err := provision.DetectProvisioner(h.Driver)
 	if err != nil {
+		crashreport.ReportCrash(err)
```

Comment:
Maybe you could just call this module `crash` and the method `Report`.  Then the invocation would be simply:

```golang
crash.Report(err)
```